### ISSUE TITLE
HW5 remaining presentataion

### ISSUE THREAD

**Issue by reshnashrestha** [ubsuny/23-Homework5G2#68]
Will there going to any presentations on HW5 for those who have not done?

**Comment by poojashresthacode** [comment]
Don't know. I think we can.

**Comment by reshnashrestha** [comment]
Yeah but we already have hw 6 and as per discussed in next class, we will be using codespace for hands-on for next hw.
